### PR TITLE
DAOS-10803 test: Update the object class and RF. (#7690)

### PR DESCRIPTION
**Pull request by knard-intel**
# Description

Backport of the patch 747fd6d updating the object class and RF for
offline_reintegration.

The oclass RP_2G8 is not adapted for the test
osa/offline_reintegration.py:offline_reintegration_oclass.  Indeed, some
shards of the ior test file could be lost during the process of the many
cycles of rebuild/reintegration. These lost shards, could eventually
makes the fds_open call of ior to fail.

More details on this issue are available in the DAOS-9313 ticket.

### CODE REVIEW

File: src/tests/ftest/osa/offline_reintegration.yaml [hunk 1]
```diff
@@ -100,7 +100,6 @@ mdtest:
 test_obj_class:
   oclass:
     - RP_2G8
-    - RP_3G6
```
Comment (rpadma2) [inline src/tests/ftest/osa/offline_reintegration.yaml:103]: You have to remove RP_2G8 since rf is set to 2.

Comment (knard-intel) [reply src/tests/ftest/osa/offline_reintegration.yaml:103]: Arghhh, thanks for the tip.
Going to fix it and relaunch the test.

**Comment by knard-intel** [comment]
> Also, I would try this commit message (I was hitting the TestIoSys failure before with RP_2G8):
> 
> Test-tag: offline_reintegrate_during_rebuild offline_reintegration_oclass online_drain_with_csum Test-repeat: 10

Still failing on the same errors :'(

**Comment by rpadma2** [comment]
Looks like the ior command was using mixed/wrong classes. Updated them and pushed a version. Let's see how it goes...

**Comment by knard-intel** [comment]
> Looks like the ior command was using mixed/wrong classes. Updated them and pushed a version. Let's see how it goes...

Seems to have fixed the problem :)
Is it ok for you, if I convert this draft PR to a plain one ?

**Review by Nasf-Fan — APPROVED** [review]

**Comment by knard-intel** [comment]
> Please add request_for_2.0.4 if you want to continue to land this?

OK, thanks for the info.

**Comment by knard-intel** [comment]
Merge for 2.0.4 was approved :)

**Comment by knard-intel** [comment]
@daos-stack/daos-gatekeeper could you merge this PR, or something is still missing ?

**Comment by knard-intel** [comment]
@daos-stack/daos-gatekeeper could you please is something missing for merging this PR ?
Thanks in advance.

**Comment by knard-intel** [comment]
Please, @daos-stack/daos-gatekeeper could you merge this PR, or tell me if it is not possible ?
If it is not possible could you tell me why.
Thanks in advance,

**Comment by mjmac** [comment]
@knard-intel: Looks like this will need to be landed by @jolivier23 whenever he is available.

**Comment by knard-intel** [comment]
> @knard-intel: Looks like this will need to be landed by @jolivier23 whenever he is available.

OK, thanks for the info :)

**Pull request merged**